libs/blocks/merch-card-collection/merch-card-collection.css

```diff
@@ -0,0 +1,3 @@
+.section[class*="-merch-cards"] .full-width {
```

Comment:
isn't the new class 'merch-card-collection'?

Reply:
these are style tokens for Milo section, e.g:  `one merch card, two merch cards, three merch cards, four merch cards`.

Reply:
got it, thx